ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,187 @@
+import os
+
+from ..constants import ENV_KEY
+from ..constants import VERSION_KEY
+from ..internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+OTEL_UNIFIED_TAG_MAPPINGS = {
+    "deployment.environment": ENV_KEY,
+    "service.name": "service",
+    "service.version": VERSION_KEY,
+}
+
+
+def _remap_otel_log_level(otel_value):
+    """Remaps the otel log level to ddtrace log level"""
+    if otel_value == "debug":
+        return "True"
+    else:
+        log.warning(
+            "ddtrace does not support otel log level '%s'. setting ddtrace to log level info.",
+            otel_value,
+        )
+        return "False"
+
+
+def _remap_otel_propagators(otel_value):
+    """Remaps the otel propagators to ddtrace propagators"""
+    accepted_styles = []
+    for style in otel_value.split(","):
+        style = style.strip().lower()
+        if style in ["b3", "b3multi", "b3single", "datadog", "tracecontext", "none"]:
+            if style == "b3single":
+                style = "b3"
+            if style not in accepted_styles:
+                accepted_styles.append(style)
+        else:
+            log.warning("Following style not supported by ddtrace: %s.", style)
+    return ",".join(accepted_styles)
+
+
+def _remap_traces_sampler(otel_value):
+    """Remaps the otel trace sampler to ddtrace trace sampler"""
+    if otel_value in ["always_on", "always_off", "traceidratio"]:
+        log.warning("Trace sampler set to %s; only parent based sampling is supported.", otel_value)
```

Comment:
should we state that we are using the parentbased version more explicitly, e.g.

```suggestion
        log.warning("Trace sampler set from %s to parentbased_%s; only parent based sampling is supported.", otel_value, otel_value)
```


which if we do that, we could even add:

```python
otel_value = f"parentbased_{otel_value}"
```

to be more explicit about it, then your checks below only need to care about the `parentbased_*` variants.

Reply:
Changed! However we're planning on supporting non-parent based samplers in a future PR, which is why we've written it this way for now

Reply:
👍🏻 either way would have been fine